packages/patterns/test/test-patterns.js

```diff
@@ -525,20 +527,13 @@ const runTests = (successCase, failCase) => {
         continue;
       }
       // This specimen is not a Key, so testing is less straightforward.
-      if (method === 'scalar' || method === 'key') {
+      if (tagIgnorantMethods.includes(method)) {
         successCase(specimen, M.not(M[method]()));
         failCase(
           specimen,
           M[method](),
           makeMessage('"[mysteryTag]"', 'mysteryTag', 'tagged'),
         );
-      } else if (method === 'null') {
-        failCase(
-          specimen,
-          M[method](),
-          simpleMethods.key('"[mysteryTag]"', 'mysteryTag', 'tagged'),
-          true,
-        );
```

Comment:
Huh. I'd forgotten that I had already made this special case. Nice to see that the rest of this PR causes this to melt away.

Wow, in four places. Nice!